Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -62,7 +62,7 @@ moduleConfigProperty :
 ;
 
 moduleConfigElement :
-    unrestrictedIdentifier whiteSpace? EQ whiteSpace? expression (COLON numberLiteral)? endOfStatement
+    unrestrictedIdentifier(DOT unrestrictedIdentifier)* whiteSpace? EQ whiteSpace? (DOLLAR | PERCENT | PLUS POW | PLUS | POW)? L_BRACE? expression R_BRACE? (COLON unrestrictedIdentifier)? endOfStatement
```

Comment:
I am rather sure this not what you want. First of all please note that `unrestrictedIdentifier` is always an identifier, which excludes all literals. Since the examples to which I adapted the grammar did have a `numberLiteral` at that position, I do not see how that option can be removed. In addition, I believe you always want to have matching braces,  right? In that case, use an alternative to either have no or both braces.

Moreover, there is a space missing in front of the first opening parenthesis.

Reply:
In addition, I think the shortcut part should go into its own rule used here. Moreover, it seems more appropriate to use an `lExpression` instead of hand rolling the member access expression at the start.

Reply:
Yep I figured `unrestrictedIdentifier` was probably not right, but I need something a little more permissive than `numberLiteral` as it needs to accept not only digits but hex digits ([0-9A-H]). I wasn't sure about exposing a HEX_DIGIT from the lexer as I didn't want it matching anything except in this very narrow case.

Not sure how to proceed.

Reply:
Please note that the hex digit also is not covered by an `unrestrictedIdentifier`.